applications/smart-distancing/ui/utils/visualization_utils.py

```diff
@@ -390,6 +391,38 @@ def visualize_boxes_and_labels_on_image_array(
     return image
 
 
+
+def checkDistance(nn_out):
+
+    # unNormalize centroid of boxes!
+    w,h = 1920,1280
+
+    for obj in nn_out:
+        centroidBox = obj["centroid"]
+        xc = centroidBox[0]*w
+        yc = centroidBox[1]*h
+        wc = centroidBox[2]*w
+        hc = centroidBox[3]*h
+        obj["centroidReal"]=[xc, yc, wc, hc]
+    distance = [] 
+    for i in range(len(nn_out)):
```

Comment:
An unnecessary loop, use the previous loop.

Reply:
previous one is removed now, however I think it wasn't unnecessary as it is needed to have all `centroidReal` boxes information to be used in next nested loop.